src/rai_bench/rai_bench/tool_calling_agent/tasks/basic.py

```
@@ -14,328 +14,222 @@
-    Validator,
 )
+from rai_bench.tool_calling_agent.mocked_ros2_interfaces import COMMON_TOPICS_AND_TYPES
 from rai_bench.tool_calling_agent.mocked_tools import (
-    MockGetObjectPositionsTool,
     MockGetROS2ImageTool,
     MockGetROS2TopicsNamesAndTypesTool,
     MockReceiveROS2MessageTool,
 )
 
 loggers_type = logging.Logger
+PROACTIVE_ROS2_EXPERT_SYSTEM_PROMPT_0_SHOT = """You are a ROS 2 expert that want to solve tasks. You have access to various tools that allow you to query the ROS 2 system.
```

Flake8 (7.2.0)

[error] line too long (171 > 79 characters)

(E501)